services/src/pro/handlers/drone_mapping.rs

```diff
@@ -312,6 +312,8 @@ async fn dataset_definition_from_geotiff(
                     spatial_reference: spatial_reference.into(),
                     measurement: Measurement::Unitless,
                     no_data_value: None, // TODO
+                    time: None,
+                    bbox: None,
```

Comment:
We could get it in principle, right?

Reply:
66619e218c2104c8577669937023a7354353f531